We don't want the build system to install to docdir for us.

--- a/meson.build	2019-10-26 23:24:34.743783753 +0200
+++ b/meson.build	2019-10-26 23:26:50.847647926 +0200
@@ -3342,15 +3342,10 @@
 # install_data('modprobe.d/systemd.conf',
 #              install_dir : modprobedir)
 #endif // 0
-install_data('LICENSE.GPL2',
-             'LICENSE.LGPL2.1',
-             'NEWS',
 #if 0 /// elogind has upgraded to markdown
 #              'README',
 #else
-             'README.md',
 #endif // 0
-             'docs/CODING_STYLE.md',
 #if 0 /// irrelevant for elogind
 #              'docs/DISTRO_PORTING.md',
 #              'docs/ENVIRONMENT.md',
@@ -3359,8 +3354,6 @@
 #              'docs/TRANSLATORS.md',
 #              'docs/UIDS-GIDS.md',
 #endif // 0
-             'src/libelogind/sd-bus/GVARIANT-SERIALIZATION',
-             install_dir : docdir)
 
 #if 0 /// UNNEEDED by elogind
 # meson.add_install_script('sh', '-c', mkdir_p.format(systemdstatedir))
